src/gui/optionsdlg.cpp

```diff
@@ -589,7 +589,7 @@ void OptionsDialog::saveOptions()
     pref->setMailNotificationSMTPUsername(m_ui->mailNotifUsername->text());
     pref->setMailNotificationSMTPPassword(m_ui->mailNotifPassword->text());
     pref->setAutoRunEnabled(m_ui->autoRunBox->isChecked());
-    pref->setAutoRunProgram(m_ui->autoRun_txt->selectedPath().trimmed());
+    pref->setAutoRunProgram(Utils::Fs::toNativePath(m_ui->autoRun_txt->selectedPath()).trimmed());
```

Comment:
It will be safer to do this when reading settings, not when writing.

Reply:
but what if user sees it in .ini file?
wouldn't it be confusing (and error prone when editing) when the path separator is different?

another idea is ditch `FileSystemPathEdit` to go back to `QLineEdit`... there is no point to use `FileSystemPathEdit` functions here.

Reply:
>but what if user sees it in .ini file?
wouldn't it be confusing (and error prone when editing) when the path separator is different?

Then it's better to do it in both reading/writing cases. As I always say, since we allow manual config editing, we can't completely trust it.
>another idea is ditch FileSystemPathEdit to go back to QLineEdit... there is no point to use FileSystemPathEdit functions here.

I don't know. But `QLineEdit` won't protect you from wrong separators.

Reply:
>As I always say, since we allow manual config editing, we can't completely trust it.
>
>But QLineEdit won't protect you from wrong separators.

In this case, we should faithfully execute what user put in, if anything goes wrong it's user's fault.

>another idea is ditch FileSystemPathEdit to go back to QLineEdit... 

If no one objects, I'll change it so.

Reply:
>In this case, we should faithfully execute what user put in, if anything goes wrong it's user's fault.

Reasonably. I like that!